scripts/github_api

```diff
@@ -86,6 +112,26 @@ if __name__ == '__main__':
                 (('--body',), dict(help='pull request body text')),
             ],
         },
+        'query_pr': {
+            'description': "Get information about a GitHub pull-request",
+            'function': query_pull_request,
+            'arguments': [
+                (('repo',), dict(
+                    help='target repository path: <repo owner>/<repo>')),
+                (('number',), dict(help='pull request number')),
+                (('--title',), dict(
+                    help='pull request title text', action='store_true')),
+                (('--state',), dict(
+                    help='pull request state, i.e. open, closed',
+                    action='store_true')),
+                (('--merged',), dict(
+                    help='whether the pull request is merged',
+                    action='store_true')),
+                (('--mergeable',), dict(
+                    help='whether the pull request can be merged',
+                    action='store_true')),
```

Comment:
Mention those options are to choose what information is retrieved. E.g. add "retrieve" or "get" to the help text.